redis/src/aio/connection_manager.rs

```diff
@@ -271,10 +289,17 @@ impl ConnectionManager {
         // Create a MultiplexedConnection and wait for it to be established
         let runtime = Runtime::locate();
 
-        let mut retry_strategy =
-            ExponentialBackoff::from_millis(config.exponent_base).factor(config.factor);
+        if config.resubsrcibe_automatically && config.push_sender.is_none() {
+            return Err((crate::ErrorKind::ClientError, "Cannot set resubsrcibe_automatically without setting a push sender to receive messages.").into());
```

Comment:
nitpick: resubsrcibe_automatically
```suggestion
            return Err((crate::ErrorKind::ClientError, "Cannot set resubscribe_automatically without setting a push sender to receive messages.").into());
```